src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/Internal/MsQuicParameterHelpers.cs

```diff
@@ -81,6 +81,35 @@ internal static unsafe void SetULongParam(MsQuicApi api, IntPtr nativeObject, ui
                 "Could not set ulong.");
         }
 
+        internal static unsafe byte GetByteParam(MsQuicApi api, IntPtr nativeObject, uint level, uint param)
+        {
+            byte* ptr = stackalloc byte[sizeof(byte)];
+            QuicBuffer buffer = new QuicBuffer()
+            {
+                Length = sizeof(byte),
+                Buffer = ptr
+            };
+
+            QuicExceptionHelpers.ThrowIfFailed(
+                api.UnsafeGetParam(nativeObject, level, param, ref buffer),
+                "Could not get byte.");
+
+            return *ptr;
+        }
+
+        internal static unsafe void SetByteParam(MsQuicApi api, IntPtr nativeObject, uint level, uint param, byte value)
+        {
+            QuicBuffer buffer = new QuicBuffer()
+            {
+                Length = sizeof(byte),
+                Buffer = &value
+            };
+
+            QuicExceptionHelpers.ThrowIfFailed(
+                api.UnsafeGetParam(nativeObject, level, param, ref buffer),
```

Comment:
typo. I think you meant UnsafeSetParam

Reply:
that's really bad, because I just copied from `SetULongParam` above, so there is another pre-existing typo. thanks.